scripts/deploy_single.sh

```
@@ -0,0 +1,444 @@
+#!/bin/bash
+set -e
+VERSION="1.0.0"
```

Shellcheck (0.10.0)

[warning] VERSION appears unused. Verify use (or export if used externally).

(SC2034)